features/support/retrospective_stub.rb

```diff
@@ -8,6 +8,8 @@ def initialize
     @calls = []
   end
 
+  # rubocop:disable Style/MethodMissingSuper
+  # rubocop:disable Style/MissingRespondToMissing
```

Comment:
You can comma-separate these:

```ruby
  # rubocop:disable Style/MethodMissingSuper, Style/MissingRespondToMissing
```

Same suggestion below.

Reply:
Oh nice, didn't realise that. I've applied.